tests/ui/test_display.py

```
@@ -0,0 +1,93 @@
+import unittest
+from unittest.mock import patch, MagicMock
+from rich.console import Console
```

Ruff (0.8.0)

`rich.console.Console` imported but unused

Remove unused import: `rich.console.Console`

(F401)